pkg/controller/clustersync/clustersync_controller.go

```diff
@@ -282,6 +282,11 @@ func (r *ReconcileClusterSync) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	// Is syncing paused?
```

Comment:
Should we consider putting this [after creating the initial ClusterSync and ClusterSyncLease](https://github.com/openshift/hive/pull/1147/files#diff-d64ff608a67db50bd6a15a35465607c1R341)?

Reply:
This looks consistent with what would happen today if the cluster was marked unreachable. I can't think of a reason we'd need a clustersync created for a new cluster that was immediately paused.